src/platform_impl/windows/dpi.rs

```diff
@@ -102,7 +104,7 @@ pub unsafe fn hwnd_dpi(hwnd: HWND) -> u32 {
         }
     } else {
         // We are on Vista or later.
-        if winuser::IsProcessDPIAware() != FALSE {
+        if IsProcessDPIAware() == true.into() {
```

Comment:
Semantic change (positive comparison to `true` instead of negative comparison to `false`)